source/operations/cert-manager/cert-manager-operator.rst

```diff
@@ -0,0 +1,239 @@
+.. _minio-certmanager-operator:
+
+=========================
+cert-manager for Operator
+=========================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+
+MinIO Operator manages the TLS certificate issuing for the services hosted in the ``minio-operator`` namespace. 
+
+This page describes how to manage the Operator's TLS certificates with :ref:`cert-manager <minio-certmanager>`.
+
+Prerequisites
+-------------
+
+- A `supported version of Kubernetes <https://kubernetes.io/releases/>`__. 
+- `kustomize <https://kustomize.io/>`__ installed
+- ``kubectl`` access to your ``k8s`` cluster
+- Completed the steps to :ref:`set up cert-manager <minio-setup-certmanager>`
+- The MinIO Operator must not yet be installed.
+
+
+1) Create a CA Issuer for the ``minio-operator`` namespace
+----------------------------------------------------------
+
+This guide **disables** the automatic generation of certificates in MinIO Operator and issues certificates using cert-manager instead.
+
+The ``minio-operator`` namespace needs to have its own certificate authority (CA), derived from the cluster's ``ClusterIssuer`` certificate created during :ref:`cert-manager setup <minio-certmanager>`.
+Create this CA certificate using cert-manager.
+
+.. important::
+
+   This CA certificate should exist *before* installing MinIO Operator.
```

Comment:
maybe this could be `must` also, it won't work otherwise I think?
```suggestion
   This CA certificate must exist *before* installing MinIO Operator.
```